package.json

```diff
@@ -9,15 +9,18 @@
     "clean-svgs": "node scripts/clean-svgs",
     "check-size": "node scripts/check-size",
     "generate-css": "node scripts/build-color-variants.js && node scripts/build-layout-scales.js",
-    "build:site": "node scripts/build-site",
+    "generate-site-data": "rm -rf _tmp_assembly && npm run copy-site-assets && npm run build-icon-json && npm run build-navigation-json",
+    "build-icon-json": "node scripts/build-icon-json.js",
+    "build-navigation-json": "node scripts/build-navigation-json.js",
+    "build:site": "npm run build:css && npm run build:js && npm run generate-site-data && batfish build",
     "build:css": "npm run generate-css && node scripts/build-css",
     "build:js": "node scripts/build-js",
     "build": "rm -rf dist && npm-run-all --parallel build:css build:js build:site",
     "lint:css": "stylelint src/*.css",
     "lint:js": "eslint .",
     "lint": "npm run lint:css && npm run lint:js",
-    "serve": "node scripts/serve",
-    "start": "npm run build:js && npm run serve",
+    "serve": "batfish serve-static",
+    "start": "npm run build:css && npm run build:js && npm run generate-site-data && batfish start",
```

Comment:
Without nodemon watching those CSS and JS files, what happens if you change one? Does the site rebuild?